scripts/doctor

```diff
@@ -39,12 +39,16 @@ if [ $found -eq 0 ] || ! ruby -e "exit Gem::Version.new(RUBY_VERSION) >= Gem::Ve
 else
     warn "grn" "   found!"
 
-    count_in_path "bundler"
+    count_in_path "bundle"
     if [ $found -eq 0 ]; then
         exitcode=1
-        warn "yel" "   bundler not found. Run 'gem install bundler' to install it it."
+        warn "yel" "   bundler not found. Run 'gem install bundler' to install it."
     else
-        warn "grn" "   found!"
+        if grep -q "Bundler version" <<< "$(bundle version)"; then
```

Comment:
```suggestion
        if grep -qi "Bundler" <<< "$(bundle version)"; then
```

nit, just in case.... it should be fine as is but I want to make the verification a little more tolerant

Reply:
It seems pretty safe, it's been a requirement from almost the beginning: https://github.com/rubygems/bundler/blob/85992a5d4a25712c7367f889e9f3589928449673/spec/commands/version_spec.rb#L23

How about "^bundler"?